client/gutenberg/editor/edit-post/hooks/components/media-upload/index.js

```diff
@@ -0,0 +1,132 @@
+/** @format */
```

Comment:
Nitpick: same thing I pointed out in some of the previous PRs - if possible I'd like to keep all of our custom additions outside of `edit-post` folder for now.

Ideally, at one point I'd like to replace all of it by `edit-post` package, and deleting the whole folder would be much trickier if he have Calypso's custom additions all over it. Let me know if that makes sense?